sleap/gui/dataviews.py

```diff
@@ -652,3 +653,21 @@
     def flags(self, index: QtCore.QModelIndex):
         """Overrides Qt method, returns flags (editable etc)."""
         return QtCore.Qt.ItemIsEnabled | QtCore.Qt.ItemIsSelectable
+    
+class CamerasTableModel(GenericTableModel):
+    """
+    Table model for for multi-view, showing which cameras 
+    are assigned to which videos
+
+    Args: 
+        obj: 'RecordingSession' which has information of cameras 
+              and paired video
+    """
+    
+    properties = ("camera", "video")
+        
+    def object_to_items(self, obj: RecordingSession):
+        return obj.linked_cameras
+    
+    def item_to_data(self, obj: RecordingSession, item: Camcorder):
+        return {'camera': item.name, 'video': obj.get_video(item).filename}
```

Comment:
Agreed, I did notice that the cameras would disappear if they did not have a linked video. I believe the above committable suggestion should make it so that all `Camcorder`s are displayed regardless whether they have a linked `Video` - which is what we want.